server/netty-server/src/main/scala/sttp/tapir/server/netty/example/NettyServerHandler.scala

```diff
@@ -0,0 +1,83 @@
+package sttp.tapir.server.netty.example
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.jdk.CollectionConverters._
+
+import io.netty.buffer.{ByteBuf, Unpooled}
+import io.netty.channel.{ChannelFutureListener, ChannelHandlerContext, SimpleChannelInboundHandler}
+import io.netty.handler.codec.http.{
+  DefaultFullHttpResponse,
+  FullHttpRequest,
+  FullHttpResponse,
+  HttpHeaderNames,
+  HttpHeaderValues,
+  HttpRequest,
+  HttpResponse,
+  HttpResponseStatus,
+  HttpUtil,
+  HttpVersion
+}
+import sttp.tapir.model.ServerResponse
+import sttp.tapir.server.netty.NettyServerInterpreter
+import sttp.tapir.server.netty.NettyServerInterpreter.NettyRoutingResult
+
+class NettyServerHandler(val handlers: List[FullHttpRequest => NettyRoutingResult])(implicit val ec: ExecutionContext)
```

Comment:
would be good to move this example to the `examples` project, and create a self-contained, stand-alone example application (like the others there)